### PR TITLE
start: update `git add`

### LINKED ISSUE #2474: start: improvements
- [x] Remove the Sharing experiments section from Get Started > Experiments

We should remove the [Sharing experiments section](https://dvc.org/doc/start/experiments#sharing-experiments) since it adds more complexity to the Get Started tutorial than there needs to be since users have to set up a Git remote as well as a DVC remote. We don't go over this at any other point in the tutorial.

If a user clones the get started repo, it works fine, but if they are following along there are some roadblocks they might run into with getting things set up. For example, I ran into a weird credentials issue with my Git remote and ended up working with @dberenbaum to do a crazy workaround.

Overall, it clouds up the purpose of this section which is running experiments.

- [x] Update the `git add` command for Get Started > Metrics, Parameters, and Plots

In the [Comparing iterations section](https://dvc.org/doc/start/metrics-parameters-plots#collecting-metrics), users can get empty Old and Change values when they run `dvc metrics diff` because the `dvc.lock` and `dvc.yaml` are missing from the `git add`.

We need to update the command to be `git add dvc.lock dvc.yaml scores.json prc.json roc.json`.

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Closes #2474

### CODE REVIEW

File: content/docs/start/data-management/metrics-parameters-plots.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ file:///Users/dvc/example-get-started/dvc_plots/index.html
 Let's save this iteration, so we can compare it later:
 
 ```dvc
-$ git add scores.json prc.json roc.json
+$ git add dvc.lock dvc.yaml scores.json prc.json roc.json
 $ git commit -a -m "Create evaluation stage"
```
Comment (jorgeorpinel) [inline content/docs/start/data-management/metrics-parameters-plots.md:154]: Ah wait I forgot about fetch.
```suggestion
$ git commit -a -m "Create evaluation stage"
$ dvc fetch data/data.xml
```
Or should it be pull?

Comment (jorgeorpinel) [reply content/docs/start/data-management/metrics-parameters-plots.md:154]: Wait... wrong PR 🤦🏼 -- deleting comment...

File: content/docs/start/data-management/metrics-parameters-plots.md [hunk 2]
```diff
@@ -150,7 +150,7 @@ file:///Users/dvc/example-get-started/dvc_plots/index.html
 Let's save this iteration, so we can compare it later:
 
 ```dvc
-$ git add scores.json prc.json roc.json
+$ git add dvc.lock dvc.yaml scores.json prc.json roc.json
```
[the remark this answers is not in the document]
Comment (jorgeorpinel) [reply content/docs/start/data-management/metrics-parameters-plots.md:153]: In fact there's a bunch of metrics and plots files (`evaluation.json`, `precision_recall.json`, `confusion_matrix.json`, and `roc.json`), while `scores.json` and `prc.json ` seem to be gone. Would this work better? Or do we want to spell out each JSON file? Feel free to commit/approve/merge @dberenbaum 🙂 

```suggestion
$ git add .gitignore dvc.lock dvc.yaml *.json
```

[the remark this answers is not in the document]
Comment (dberenbaum) [reply content/docs/start/data-management/metrics-parameters-plots.md:153]:
```suggestion
$ git add .gitignore dvc.yaml dvc.lock evaluation.json evaluation
```

I think we can mirror https://github.com/iterative/example-repos-dev/blob/cfa4ca4d4097cab26b2921026dfb3a41c70fadaf/example-get-started/generate.sh#L155.

In general, we should either manually or automatically check for updates in these code blocks whenever we deploy an update to that repo.

**Review by dberenbaum — APPROVED** [review]

**Pull request merged**